charts/rancher-vsphere-csi/values.yaml

```diff
@@ -120,12 +125,12 @@ global:
     systemDefaultRegistry: ""
 
 versionOverrides:
-  - constraint: ">= 1.24 < 1.26"
+  - constraint: ">= 1.24"
```

Comment:
This is being updated in https://github.com/rancher/vsphere-charts/pull/35

Reply:
Updated to match line 123